GabrielaRamirez/EjerciciosHackerrank/src/basics/StdinI.java

```diff
@@ -0,0 +1,15 @@
+package basics;
+import java.util.Scanner;
+
+public class StdinI {
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        int a = scan.nextInt();
```

Comment:
Remember, variable names and parameter names must be meaningful:
https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names

Reply:
i'll change